QModManager/Patching/Patcher.cs

```diff
@@ -55,7 +55,7 @@ internal static void Patch()
                     Logger.Fatal("There was an error with the QMods directory");
                     Logger.Fatal("Please make sure that you ran Subnautica from Steam/Epic/Discord, and not from the executable file!");
 
-                    Dialog.Show("A fatal error has occurred. QModManager could not be initialized.", Dialog.Button.SeeLog, Dialog.Button.Disabled, false, true);
+                    Dialog.Show("A fatal error has occurred. QModManager could not be initialized.", Dialog.Button.Close, Dialog.Button.Disabled, false, true);
```

Comment:
The last parameter of this function is `cannotClose`. If this parameter is set to true, the dialog cannot be closed, so a button of `Close` doesn't make sense here.

Reply:
Everything to do with the dialog popup I leave to you